docs/tutorials/google/identifying_hardware_changes.ipynb

```diff
@@ -720,7 +720,7 @@
     "\n",
     "You've selected better candidate qubits for your circuit, based on updated information about the device. What else can you do for further improvements? \n",
     "\n",
-    "*    You need to map your actual circuit's logical qubits to your selected hardware qubits. This is in general a difficult problem, and the best solution can depend on the specific structure of the circuit to be run. Take a look at the [Qubit Picking with Loschmidt Echoes](./echoes.ipynb) tutorial, which estimates the error rates of gates for your specific circuit. Also, consider [Best Practices#Qubit picking](../../google/best_practices.md#qubit-picking) for additional advice on this. \n",
+    "*    You need to map your actual circuit's logical qubits to your selected hardware qubits. This is in general a difficult problem, and the best solution can depend on the specific structure of the circuit to be run. Take a look at the [Qubit Picking with Loschmidt Echoes](./echoes.ipynb) tutorial, which estimates the error rates of gates for your specific circuit. Also, consider [Best Practices#Qubit picking](../../google/best_practices.ipynb#qubit-picking) for additional advice on this.\n",
```

Comment:
ipynb header link doesn't seem to work for me (links to the page, but not the header)

Reply:
Fixed the link so it works locally on the jupyter notebooks. I think the docs infra will replace a dash with an underscore so it also works on the website (based on my observation of existing links, where the link in the code uses a dash but if you click on the website, it is replaced with an underscore with works on the website). 

I'll watch out to verify once this is submitted.